src/templates/items/parts/item-duration.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="form-group input-select">
+    <label>{{localize "SFRPG.EffectDurationTypesLabel"}}</label>
+    <div class="form-fields">
+        <input type="text" name="system.activeDuration.value" value="{{itemData.activeDuration.value}}" data-dtype="Number" placeholder="-"/>
+        <select name="system.activeDuration.unit">
+            {{#select itemData.activeDuration.unit}}
+            <option value=""></option>
+            {{#each config.effectDurationTypes as |name key|}}
+            <option value="{{key}}">{{localize name}}</option>
+            {{/each}}
+            {{/select}}
+        </select>
+    </div>
+</div>
+<div class="form-group input-select">
+    <label>{{localize "SFRPG.EffectEndTypesLabel"}}</label>
```

Comment:
I prefer something like "Ends on" for the label (which would require "on turn start" to become "turn start")